content/en/docs/concepts/cluster-administration/node-autoscaling.md

```diff
@@ -0,0 +1,264 @@
+is more straightforward than non-empty ones, and autoscalers often have optimizations
+designed specifically for consolidating empty nodes.
+
+Removing non-empty Nodes during consolidation is disruptive - the Pods running on them
+are terminated, and possibly have to be recreated (for example by a Deployment). However, all such
+recreated pods should be able to schedule on existing Nodes in the cluster, or the
+replacement Nodes provisioned as part of consolidation. No pods should
+become pending as a result of consolidation.
+
+{{< note >}}
+Consolidation is prone to certain race conditions. For example, if a new Pod is
+created right after a Node is consolidated, it can get scheduled on an existing Node
+in the cluster instead of the Pods recreated because of consolidation - leaving
+these Pods pending. This scenario is usually automatically recovered by provisioning.
+{{< /note >}}
+
+Depending on the choice of autoscaler, consolidation can also be triggered by
+other conditions (for example the time elapsed since a Node was created), in order
+to optimize different properties (for example the maximum lifespan of Nodes in a
+cluster).
+
+The details of how consolidation is performed depend on the configuration of a
+given autoscaler. Consult [Configure workloads for autoscaling](#workload-config)
+for common workload-level configuration options.
+
+{{< note >}}
+Consolidation is formerly known as _scale-down_ in Cluster Autoscaler.
+{{< /note >}}
+
+## Autoscalers {#autoscalers}
+
+The functionalities described in previous sections are provided by Node _autoscalers_.
+In addition to the Kubernetes API, autoscalers also need to interact with cloud provider
+APIs to provision and consolidate Nodes. This means that they need to be explicitly
+integrated with each supported cloud provider. The performance and feature set of
+a given autoscaler can differ between cloud provider integrations.
+
+{{< mermaid >}}
+graph TD
+    na[Node autoscaler]
+    k8s[Kubernetes]
+    cp[Cloud Provider]
+
+    k8s --> |get Pods/Nodes|na
+    na --> |drain Nodes|k8s
+    na --> |create/remove VMs|cp
+    cp --> |get VMs|na
+
+    classDef white_on_blue fill:#326ce5,stroke:#fff,stroke-width:4px,color:#fff;
+    classDef blue_on_white fill:#fff,stroke:#bbb,stroke-width:2px,color:#326ce5;
+    class na blue_on_white;
+    class k8s,cp white_on_blue;
+{{</ mermaid >}}
+
+### Autoscaler implementations
+
+Cluster autoscaler and Karpenter are the two Node autoscalers currently supported by
+[sig-autoscaling](https://github.com/kubernetes/community/tree/master/sig-autoscaling).
+
+From the perspective of a cluster user, both autoscalers should provide a similar Node autoscaling
+experience. Both will provision new Nodes for unschedulable Pods, and both will
+consolidate the Nodes that are no longer optimally utilized.
+
+The most of the difference between the autoscalers is in how they're set up and
```

Comment:
I think we've already stated the first part of this sentence already in the doc. We could add a simple statement here "Different autoscalers may provide features outsice the Node autoscaling scope described on this page, and those additional features may differ between them."